spec.md

```diff
@@ -1155,18 +1293,36 @@ compilable to non-parameterized Verilog.
 
 ## Passive Types
 
-It is inappropriate for some circuit components to be declared with a type that
-allows for data to flow in both directions. For example, all sub-elements in a
-memory should flow in the same direction. These components are restricted to
-only have a passive type.
+Stateful elements, such as registers and memories, may contain data of aggregate types.
```

Comment:
is "data" a precisely defined term?

Reply:
No.

The style that I'm going to start moving things towards is that the first sentence/paragraph of each section is informal motivation. A precise explanation of what "data" is or what it means to "contain data" would require we have rules regarding simulation of a FIRRTL design.